Taskfile.yml

```diff
@@ -104,6 +104,8 @@ tasks:
       FILES_TO_DELETE:
         sh: ls -1 "{{.WORKSPACE_DIR}}" | grep -v -e 'main.tf' -e 'backend.tf'
     cmds:
+      # TODO: this is broken because of line spaces! (the \n character)
```

Comment:
@davidbloss noticed this bug, could it be a difference in the shells we are using? I'm using zsh/oh-my-zsh and the macOS version of `ls`.

Reply:
Can we change the command such that it just deletes the `tfstate` file and backup file? I think it's better if the file doesn't touch any `.tf` files at all, the user can clean those manually if they want to.